store/issue.go

```diff
@@ -538,13 +538,34 @@ func (*Store) findIssueImpl(ctx context.Context, tx *sql.Tx, find *api.IssueFind
 		args = append(args, *v)
 		args = append(args, *v)
 	}
+	// If no specified project, filter out issues belonging to archived project
```

Comment:
Shall we do it in FindIssue instead?

Reply:
The existing `IssueFind` doesn't allow us to do it in FindIssue.